pkg/capture/capture.go

```diff
@@ -203,7 +212,7 @@ func (c *Capture) rotate(ctx context.Context) (agg *hashmap.AggFlowMap) {
 //
 // process keeps running until Close is called on its capture handle or it encounters
 // a serious capture error
-func (c *Capture) process(ctx context.Context) <-chan error {
```

Comment:
Context not needed anymore since the routine doesn't log and does all the termination logic via the `CaptureManager`.